src/transformers/training_args.py

```diff
@@ -1355,6 +1361,18 @@ class TrainingArguments:
             "help": "Whether to recursively concat inputs/losses/labels/predictions across batches. If `False`, will instead store them as lists, with each batch kept separate."
         },
     )
+    eval_preserve_model_output: bool = field(
+        default=False,
+        metadata={
+            "help": "Whether to return the outputs of the model as `ModelOutput` objects during evaluation, or to return the outputs unpacked as a tuple."
+        },
+    )
+    eval_numpify_tensors: bool = field(
+        default=True,
+        metadata={
+            "help": "Whether to convert the tensors to numpy arrays before passing them to the `compute_metrics` function and returning from `Trainer.evaluation_loop`."
+        },
+    )
```

Comment:
With so many `eval_` args now, we may want to consider something like an `EvaluationConfiguration` perhaps cc @amyeroberts